content/en-us/reference/engine/classes/VoiceChatService.yaml

```
@@ -16,15 +17,16 @@ deprecation_message: ''
 properties:
   - name: VoiceChatService.EnableDefaultVoice
     summary: |
-      Indicates whether or not the place uses the default voice manager.
+      Controls whether or not the default spatial voice chat setup will be used.
     description: |
-      Indicates whether or not the place uses the default voice manager. This is
-      enabled by default.
+      Controls whether or not the default spatial voice chat setup will be used.
 
-      When enabled, the default voice manager adds all voice-eligible users to
-      voice chat as they join the place. When disabled, the place doesn't use
-      the default voice manager, and users are not added to voice chat when they
-      join the place.
+      When enabled, each voice-eligible player can be heard as though they were speaking through their character.
+      If `Class.VoiceChatService.EnableDefaultVoice` is `true`, and `Class.VoiceChatService.UseAudioApi` is `Enabled`,
+      then this setup is represented and controlled by instance types from the Audio API:
+      - An `Class.AudioDeviceInput` will be created and parented to each voice-eligible `Class.Player`
+      - An `Class.AudioEmitter` will be created and parented to each voice-eligible player's `Class.Player.Character`
+      - An `Class.AudioListener` will be created and parented to `Class.Workspace.CurrentCamera`
```

And we can get into the specifics here

```suggestion
      When enabled, each voice-eligible player can be heard as though they were speaking through their character.
      
      The behavior of the default voice setup depends on `Class.VoiceChatService.UseAudioApi`. 
      If `Class.VoiceChatService.UseAudioApi` is `Enabled`, then this setup is represented and controlled by instance types from the Audio API. Specifically:
      - An `Class.AudioDeviceInput` will be created and parented to each voice-eligible `Class.Player`
      - An `Class.AudioEmitter` will be created and parented to each voice-eligible player's `Class.Player.Character`
      - An `Class.AudioListener` will be created and parented to `Class.Workspace.CurrentCamera`
      
      If `Class.VoiceChatService.UseAudioApi` is `Disabled`, then this setup is made by an internal-only system. 
      Using this system, disabling the default voice setup effectively disables voice altogether.
```